storage-operator/go.mod

```diff
@@ -1,7 +1,6 @@
 module github.com/scality/metalk8s/storage-operator
 
 require (
-	github.com/NYTimes/gziphandler v1.0.1 // indirect
```

Comment:
shouldn't commit that

Reply:
Oh indeed `git commit -a` MB :stuck_out_tongue: